packages/ember-glimmer/lib/environment.js

```diff
@@ -268,8 +268,9 @@ export default class Environment extends GlimmerEnvironment {
 
   getComponentDefinition(path, parentMeta) {
     let name = path[0];
+    let owner = !!(parentMeta && parentMeta.owner) ? parentMeta.owner : this.owner;
```

Comment:
I believe that this may be a bug in glimmer.  The `parentMeta` should always be present.  

I believe that [these lines](https://github.com/tildeio/glimmer/blob/c0a92795b2df68bfed674683c2ae9c3d2b633835/packages/glimmer-runtime/lib/environment.ts#L235-L239) should be changed to include `blockMeta`.

I'm happy to iterate on this in a follow-up though...